docs/build/wasp-evm/v1.0.0-rc.6/docs/reference/core-contracts/accounts.md

```diff
@@ -60,6 +60,17 @@ amount must be able to cover the L1 storage deposit. Otherwise, it will fail.
 Transfers the specified allowance from the sender's L2 account to the given L2 account on
 the chain.
 
+:::note
+When a transfer is made into an EVM account, an EVM tx will be created on the EVM side from the zero address (0x0000...) to the target account.
+Information about what is being transferred will be encoded in the that transction's data using the following format:
```

Comment:
The format has to be:

```txt
:::note

Text

:::
```
Don't forget the new lines between the text and `:::` https://docusaurus.io/docs/markdown-features/admonitions

```suggestion
:::note

When a transfer is made into an EVM account, an EVM tx will be created on the EVM side from the zero address (0x0000...) to the target account.
Information about what is being transferred will be encoded in the that transction's data using the following format:

:::
```